pycqed/analysis_v2/base_analysis.py

```diff
@@ -646,9 +646,11 @@ def extract_data(self):
             self.raw_data_dict = tuple(temp_dict_list)
             if 'TwoD' not in self.options_dict:
                 if not all(twod_list):
-                    raise ValueError('Not all measurements have the same '
-                                     'number of sweep dimensions.')
-                self.options_dict['TwoD'] = twod_list[0]
+                    raise log.info('Not all measurements have the same '
```

Comment:
`raise log.info` does not look good to me ;-)

Reply:
fixed it now